.github/workflows/test-core.yaml

```diff
@@ -0,0 +1,152 @@
+name: Core CI Tests
+on:
+  pull_request:
+    branches:
+      - main
+    paths-ignore:
+      - 'README.md'
+      - 'CHANGELOG.md'
+      - '.changelog/*'
+      - '.tours/*'
+      - 'contributing/*'
+      - 'demo/*'
+      - 'pkg/*'
+      - 'scripts/*'
+      - 'terraform/*'
+      - 'ui/*'
+      - 'website/*'
+  push:
+    branches-ignore:
+      - main
+      - release-**
+    paths-ignore:
+      - 'README.md'
+      - 'CHANGELOG.md'
+      - '.changelog/*'
+      - '.tours/*'
+      - 'contributing/*'
+      - 'demo/*'
+      - 'pkg/*'
+      - 'scripts/*'
+      - 'terraform/*'
+      - 'ui/*'
+      - 'website/*'
+env:
+  GO_VERSION: 1.17.7
```

Comment:
We probably should update [`scripts/update_golang_version.sh`](https://github.com/hashicorp/nomad/blob/main/scripts/update_golang_version.sh) to update this file as well.

Reply:
good catch, fixed!